### PR TITLE
15959 - UXA fix - Relocate click box and Add Account Creation Process for GOVN

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:*
https://github.com/bcgov/entity/issues/15959

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-api/migrations/versions/2024_06_05_7bf8428e568e_.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""empty message
```
Comment (seeker25) [inline auth-api/migrations/versions/2024_06_05_7bf8428e568e_.py:1]: """empty message

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2857-3xrbyydp.web.app

File: auth-web/src/stores/org.ts [hunk 2]
```diff
@@ -155,8 +155,8 @@ export const useOrgStore = defineStore('org', () => {
     return state.currentOrganization?.isBusinessAccount === true
   })
 
-  const isGovnOrg = computed<boolean>(() => {
-    return state.originAccessType === AccessType.GOVN
+  const isGovnGovmOrg = computed<boolean>(() => {
+    return state.originAccessType === (AccessType.GOVN || AccessType.GOVN)
```
Comment (seeker25) [inline auth-web/src/stores/org.ts:159]: `[AccessType.GOVN || AccessType.GOVM].includes(state.originAccessType)`

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2857-3xrbyydp.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2857-3xrbyydp.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2857-3xrbyydp.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2857-3xrbyydp.web.app

**Pull request merged**